tools/package_tools/mmdeploy_builder.py

```diff
@@ -173,6 +174,40 @@ def build_mmdeploy(cfg, mmdeploy_dir, dist_dir=None):
     _call_command(bdist_cmd, mmdeploy_dir)
 
 
+def build_mmdeploy_python(python_executable, cfg, mmdeploy_dir):
+    cmake_envs = cfg.get('cmake_envs', dict())
+
+    args = [f'-D{k}={v}' for k, v in cmake_envs.items()]
+    args.append(
+        f'-DMMDeploy_DIR={mmdeploy_dir}/build/install/lib/cmake/MMDeploy')
+
+    if sys.platform == 'win32':
+        build_cmd = 'cmake --build . --config Release -- /m'
+        pass
+    else:
+        args.append(f'-DPYTHON_EXECUTABLE={python_executable}')
```

Comment:
move `args.append(f'-DPYTHON_EXECUTABLE={python_executable}')` upwards to let both windows and linux use it.